### DISCUSSION TITLE
AlpineJS support?

### DISCUSSION THREAD

**Discussion by that0n3guy** [tailwindlabs/headlessui#662] [Ideas]
Its been a while since I've seen mention of alpine in here.   Back in october there were a couple mentions of it being supported soon.

Is that still the case?  Is Alpinejs support coming?

**Comment by tylerwiegand** [comment]
My guess is no because I remember Caleb Porzio mentioning that was a thing he wanted to do next, premium UI elements for alpine after v3 stabilized. Im a bit bummed about it because its one of the reasons i bought TailwindUI :-/ hope im wrong!

**Reply by chrisdicarlo** [reply]
Same here.  I'm really not keen about the prospect of premium UI elements from the Alpine project - to me, Alpine is equivalent to Vue/React and I want to be able to mix and match using the Tailwind UI designs, not have to try and integrate different design systems/styles to make the design of MY site cohesive.  Might just be time to ditch Alpine in favour of Vue just to be able to copy and paste without having to constantly re-implement the same logic.  Bummer, indeed!
